ttyclock.h

```diff
@@ -78,7 +78,7 @@ typedef struct
           Bool utc;
           Bool box;
           Bool noquit;
-          char *format;
+          char format[512];
```

Comment:
it seems we gratuitously changed the length of the format string from 100 to 512 here.

Reply:
Should I revert to 100?

Reply:
Yes, I think it's large enough for a date.